Integrations/integration-QRadar.yml

```diff
@@ -42,296 +38,554 @@ configuration:
   defaultvalue: "true"
   type: 8
   required: false
-- display: Incident type
-  name: incidentType
-  defaultvalue: ""
-  type: 13
-  required: false
 - display: Use system proxy settings
   name: proxy
   defaultvalue: "true"
   type: 8
   required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
 script:
   script: |-
-    // The command input arg holds the command sent from the user.
-    var username = params.credentials ? params.credentials.identifier : '';
-    var password = params.credentials ? params.credentials.password : '';
-    var token = params.token;
-    if (!token && !password) {
-        throw 'Either credentials or auth token should be provided.'
-    }
-    var server = params.server+'/';
-    var insecure = params.insecure;
-    var query = params.query;
-    var offensesPerCall = parseInt(params.offensesPerCall || 50);
-    offensesPerCall = offensesPerCall > 50 ? 50 : offensesPerCall;
-
-    var createIncidentFromOffense = function(incident) {
-        var keys = Object.keys(incident);
-        var labels = [];
-        var occured = (new Date(incident.start_time)).toISOString();
-        for (var i = 0; i<keys.length; i++) {
-            labels.push({'type': keys[i], 'value': String(incident[keys[i]])});
-        }
+    import sys, os, requests, traceback, datetime, json
+    from requests.exceptions import HTTPError
+
+    ''' GLOBAL VARS '''
+    SERVER = demisto.params()['server'][:-1] if demisto.params()['server'].endswith('/') else demisto.params()['server']
+    USERNAME = demisto.params().get('credentials').get('identifier')
+    PASSWORD = demisto.params().get('credentials').get('password')
+    TOKEN = demisto.params().get('token')
+    USE_SSL = not demisto.params().get('insecure', False)
+    CUSTOM_HEADERS = demisto.args().get('headers')
```

Comment:
arg shouldn't be global var.
Not all the commands has `headers` argument I suppose